numpy/linalg/linalg.py

```diff
@@ -1947,6 +1947,12 @@ def lstsq(a, b, rcond=-1):
         lrwork = int(rwork[0])
         liwork = iwork[0]
 
+        if any(size == 0 for size in (lwork, lrwork, liwork)):
+            raise RuntimeError(
+                'LAPACK did not compute workspace sizes: len(work) = {}, len(rwork) = {}, len(iwork) = {}'
+                .format(lwork, lrwork, liwork)
+            )
```

Comment:
This is a known and fixed (9 Jun 2010) bug in LAPACK: http://icl.cs.utk.edu/lapack-forum/archives/lapack/msg00899.html/.

However, this fails on Appveyor. What version of LAPACK is being used, and do we really need to support a version released before python 2.7.0?